src/main/java/picard/arrays/illumina/InfiniumEGTFile.java

```diff
@@ -141,22 +131,11 @@ private void readFileData() throws IOException {
 
     protected void initializeArrays(int numCodes) {
 
-        nAA = new int[numCodes];
-        nAB = new int[numCodes];
-        nBB = new int[numCodes];
-
-        devRAA = new float[numCodes];
-        devRAB = new float[numCodes];
-        devRBB = new float[numCodes];
-        meanRAA = new float[numCodes];
-        meanRAB = new float[numCodes];
-        meanRBB = new float[numCodes];
-        devThetaAA = new float[numCodes];
-        devThetaAB = new float[numCodes];
-        devThetaBB = new float[numCodes];
-        meanThetaAA = new float[numCodes];
-        meanThetaAB = new float[numCodes];
-        meanThetaBB = new float[numCodes];
+        n = new int[numCodes][InfiniumVcfFields.GENOTYPE_VALUES.values().length];
+        devR = new float[numCodes][InfiniumVcfFields.GENOTYPE_VALUES.values().length];
+        meanR = new float[numCodes][InfiniumVcfFields.GENOTYPE_VALUES.values().length];
+        devTheta = new float[numCodes][InfiniumVcfFields.GENOTYPE_VALUES.values().length];
+        meanTheta = new float[numCodes][InfiniumVcfFields.GENOTYPE_VALUES.values().length];
```

Comment:
extract `InfiniumVcfFields.GENOTYPE_VALUES.values().length` to a static constant

Reply:
Done